src/components/SubForm.jsx

```diff
@@ -0,0 +1,86 @@
+import { getByTitle } from '@testing-library/react';
+import React from 'react';
+import RoundedButton from './RoundedButton';
+import iconFile from '../assets/images/iconFile.svg';
+import iconTrash from '../assets/images/iconTrash.svg';
+
+import { useState, useEffect, useContext } from 'react';
+import { Link } from 'react-router-dom';
+import axios from 'axios';
+import HomeButton from '../components/HomeButton';
+import TextImageInput from './TextImageInput';
```

Comment:
```suggestion
import React from 'react';
import RoundedButton from './RoundedButton';
import iconFile from '../assets/images/iconFile.svg';
import iconTrash from '../assets/images/iconTrash.svg';

import { useState, useEffect, useContext } from 'react';
import axios from 'axios';
```
- Importações não usadas;